apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -234,6 +234,12 @@ type NetworkPublishing struct {
 	//
 	// See: https://kubernetes.io/docs/concepts/services-networking/service/#nodeport
 	//
+	// NOTE:
+	// When provisioning an Envoy `NodePortService`, use the Listeners' port numbers to populate
```

Comment:
nit: to make it clear we're talking about the Gateway Listener configuration not an Envoy Listener
```suggestion
	// When provisioning an Envoy `NodePortService`, use Gateway Listener port numbers to populate
```